README.rst

```diff
@@ -29,6 +29,22 @@ system actions.
 
 The RMA supports all backends implementing the 1.0.x `API's voor Zaakgericht Werken`_.
 
+Quickstart
+==========
+
+A `docker-compose-quickstart.yml`_ is provided to get up and running quickly. To run the container:
+
+    .. code:: shell
+
+        $ git clone https://github.com/maykinmedia/record-management-app.git
```

Comment:
Now when you use the image from docker hub you don't need to clone the repo
You can just download `docker-compose-quickstart.yml` with `wget` or `curl`